src/main/java/jacchm/footballapp/mapping/mapper/StandingsInputMapper.java

```diff
@@ -0,0 +1,229 @@
+package jacchm.footballapp.mapping.mapper;
+
+import jacchm.footballapp.mapping.dto.*;
+import jacchm.footballapp.model.entity.*;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/* TODO: Simplify this interface. Mapstruct was not working correctly - some of the fields in the mapped objects were
+*        missing. Default methods have been added to map classes correctly. */
+
+@Mapper(componentModel = "spring")

```

Comment:
jeśli masz jakieś nulle to zamiast default lepiej użyć parametru uses
`@Mapper(componentModel = "spring", uses = TableInputMapper.class, injectionStrategy = InjectionStrategy.CONSTRUCTOR) - powinno wymusić wstrzyknięcie i użycie innego mappera - jeśli trzeba więcej niż jeden inny mapper to można podać listę